src/main/scala/com/intenthq/action_processor/integrations/feeds/CSVFeed.scala

```diff
@@ -0,0 +1,28 @@
+package com.intenthq.action_processor.integrations.feeds
+
+import java.io.StringReader
+
+import cats.effect.{IO, Resource}
+import cats.implicits.catsSyntaxApplicativeId
+import de.siegmar.fastcsv.reader.CsvReader
+import fs2.Stream
+
+import scala.jdk.CollectionConverters._
+
+trait CSVFeed[O] extends Feed[Iterable[String], O] {
+
+  protected lazy val csvReader: CsvReader = new CsvReader
+
+  protected def rows: Stream[IO, String]
+
+  private def csvParse(line: String): IO[Iterable[String]] =
+    Resource.fromAutoCloseable(IO.delay(new StringReader(line))).use { sr =>
+      Option(csvReader.parse(sr).nextRow())
+        .map(_.getFields.asScala)
+        .getOrElse(collection.mutable.Buffer.empty[String])
```

Comment:
Can we replace `collection.mutable.Buffer.empty[String]` by `Iterable.empty[String]`?